.github/workflows/release.yml

```
@@ -0,0 +1,56 @@
+            artifact_name: askman-macos-x86_64
+          - os: macos-latest
+            target: aarch64-apple-darwin
+            artifact_name: askman-macos-aarch64
+          # (Temporarily disabled windows build as requested)
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.11)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)